charts/common/values.yaml

```diff
@@ -135,25 +135,27 @@ ingress:
   annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
-  # -- A list of hosts for the Ingress
+  # -- Hostname used for the default ingress rule. If .Values.ingress.rules is set, this host is not used.
+  host: chart-example.local
```

Comment:
would this also work if a service needs more than one domain configured?

Reply:
For any custom use case, the `.Values.ingress.rules` need to be updated. The default rule for most common use cases: forward all traffic from one domain to the HTTP port.